src/Components/IndexScene/IndexScene.tsx

```diff
@@ -62,7 +62,7 @@ export class IndexScene extends SceneObjectBase<IndexSceneState> {
       $timeRange: state.$timeRange ?? new SceneTimeRange({}),
       $variables:
         state.$variables ??
-        getVariableSet('grafanacloud-logs' ?? getLastUsedDataSourceFromStorage(), state.initialFilters),
+        getVariableSet(getLastUsedDataSourceFromStorage() ?? 'grafanacloud-logs', state.initialFilters),
```

Comment:
I saw this in the PR from a few days ago but it didn't click.

Reply:
I made that PR and it didn't click.